src/modules/solvers/QiskitQAOA.py

```diff
@@ -122,7 +127,7 @@ def get_parameter_options(self) -> dict:
             },
             "optimizer": {
                 "values": ["POWELL", "SPSA", "COBYLA"],
-                "description": "Which Qiskit solver should be used?"
+                "description": "Which Qiskit solver should be used? Warning: When using the IBM Eagle Device you should not use the SPSA optimizer, since it is not suited for only on evaluation!"
```

Comment:
Maybe there is a way to only raise the warning if the IBM Eagle device is selected.